docs/cloud-tutorials/application-management.md

```diff
@@ -11,25 +11,32 @@ In this guide, you'll learn how to manage applications in DBOS Cloud.
 
 To deploy your application to DBOS Cloud, run this command in its root directory:
 
-```
+```shell
 npx dbos-cloud app deploy
 ```
 
 Your application is deployed using the name in its `package.json`.
-Application names should be between 3 and 30 characters and must contain only lowercase letters and numbers, dashes (`-`), and underscores (`_`).
+Application names should be between 3 and 30 characters and must contain only lowercase letters and numbers, dashes (`-`), and underscores (`_`). Application names are unique within an [organization](#users-and-organizations).
 
-When you deploy an application for the first time, the command prompts you to choose which [database instance](../cloud-tutorials/database-management.md) to connect your app to, or to provision one if you have none.
+When you deploy an application for the first time, the command prompts you to choose which [database instance](../cloud-tutorials/database-management.md) to connect your app to, or to provision one if you have none. Note this database instance (server) is different from the database name your application use (the `app_db_name` field in [`dbos-config.yaml`](../api-reference/configuration#database)).
 
 :::tip
-You can specify a particular database instance through the `-d <database-instance-name>`.
-To enable time travel for your application, specify `--enable-timetravel`.
+* You can specify a particular database instance through the `-d <database-instance-name>`.
+* During the first deployment, you can enable time travel for your application with `--enable-timetravel`. You can delete and re-deploy an existing application to enable time travel.
```

Comment:
Just for my understanding: If I do not enable timetravel at first, it is not enough to just re-depoy with `--enable-timetravel`? I have to delete the application and re-deploy?

Reply:
yup you have to delete + redeploy